### PR TITLE
Create fr_fr.json

### PR DESCRIPTION
**Pull request by Aarrn33**
I added a french localisation

### CODE REVIEW

File: src/main/resources/assets/mantle/lang/fr_fr.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  "command.mantle.dump_tag.success": "L'étiquette %s de '%s' a été ajouté au registre de la console",
+  "command.mantle.dump_tag.read_error": "Une erreur a eu lieu en lisant l'étiquette %s de '%s':",
+  "command.mantle.dump_all_tags.success": "Toutes les étiquettes ont été mise dans '%s'",
+  "command.mantle.dump_all_tags.type_success": "Toutes les étiquettes %s de '%s' ont été mise",
+
+  "command.mantle.tags_for.success": "Les étiquettes %s pour '%s':",
+
+  "command.mantle.tag.empty": "Vide",
+  "command.mantle.tag_collection.not_found": "Collecirion '%s' d'étiquettes inconnue",
```
Comment (Aspyro) [inline src/main/resources/assets/mantle/lang/fr_fr.json]: Typo : ~~Collecirion~~ -> Collection

File: src/main/resources/assets/mantle/lang/fr_fr.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  "command.mantle.view_tags.success": "Valeurs pour l'étiquette '%s' de %s:",
+  "command.mantle.view_tags.not_found": "Valeur %s inconnue du registre '%s'",
+  "command.mantle.view_tags.no_held_block": "L'objet tenu n'est pas un bloc",
+  "command.mantle.view_tags.no_held_entity": "L'objet tenu n'est pas un oeuf d'apparition",
+  "command.mantle.view_tags.no_held_potion": "L'objet tenu n'est pas une potion",
+  "command.mantle.view_tags.no_held_fluid": "Incapable d'obtenir un fluide à partir de l'objet tenu",
+  "command.mantle.view_tags.no_held_enchantment": "L'objet tenu n'est pas enchanté",
+  "command.mantle.view_tags.no_targeted_entity": "Pas d'entité dans la zone",
+  "command.mantle.view_tags.no_targeted_tile_entity": "Pas de fluide dans la zone",
```
Comment (Aspyro) [inline src/main/resources/assets/mantle/lang/fr_fr.json]: Translation "dans la zone" (*in the zone*) would be more precise if switched with "à portée" (*in range*).

File: src/main/resources/assets/mantle/lang/fr_fr.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "command.mantle.view_tags.no_tags": "Pas d'étiquette",
```
Comment (Aspyro) [inline src/main/resources/assets/mantle/lang/fr_fr.json]: Linked with keys `"command.mantle.view_tags.(...)"` but should be `"command.mantle.tags_for.(...)"`.

File: src/main/resources/assets/mantle/lang/fr_fr.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+  "command.mantle.dump_tag.success": "L'étiquette %s de '%s' a été ajouté au registre de la console",
+  "command.mantle.dump_tag.read_error": "Une erreur a eu lieu en lisant l'étiquette %s de '%s':",
+  "command.mantle.dump_all_tags.success": "Toutes les étiquettes ont été mise dans '%s'",
+  "command.mantle.dump_all_tags.type_success": "Toutes les étiquettes %s de '%s' ont été mise",
```
Comment (Aspyro) [inline src/main/resources/assets/mantle/lang/fr_fr.json]: Literal (and better) translation would be `"Toutes les étiquettes de %s ont été mises dans '%s'"`

**Review by Aspyro — CHANGES_REQUESTED** [review]

**Comment by Aspyro** [comment]
I believe now it can be merged. It fully translates the file `en_us.json`.

**Comment by Aarrn33** [comment]
Thanhs

**Pull request merged**